lib/rules/cli/4.4.yaml

```diff
@@ -1136,6 +1136,13 @@
     :sync_config: --sync-config=<value>
     :type: --type=<value>
     :whitelist: --whitelist=<value>
+:oadm_inspect:
+  :cmd: oc adm inspect
+  :options:
+    :all_namespaces: --all-namespaces=<value>
```

Comment:
@zhouying7780  `all_namespaces` is a already defined, so I think we can take it out.

Reply:
@zhouying7780 please ignore comment...all_namespaces is apparently not defined as a global option...perhaps it should.

Reply:
That was my initial reaction. But `oc options` does not return it. So it should not be in globals.